build.gradle.kts

```diff
@@ -5,6 +5,7 @@ import com.github.vlsi.gradle.properties.dsl.props
 plugins {
     `java-library`
     application
+    id("org.sonarqube") version "3.0"
```

Comment:
The version should go into `gradle.properties` together with the id declaration in `settings.gradle.kts`